### ISSUE TITLE
Documentation in twisted.words.im

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6481]
|<img alt="Upasana's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Upasana reported|
|-|-|
|Trac ID|trac#6481|
|Type|enhancement|
|Created|2013-04-26 21:52:46Z|

```
W9002:  1,0: Missing a reference to test module in header
W9201:  1,0: The opening/closing of docstring should be on a line by themselves
```

Attachments:

* [bug_6481.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3ed/3ed0f159b311e7c68bb73578666d93bb3185bb24/be5dc51dd38bbfb8cf6358fd06cc79627a86be42.patch) (366 bytes) - added by Upasana on 2013-04-26 21:54:21Z - 
* [bug_6481_2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3ed/3ed0f159b311e7c68bb73578666d93bb3185bb24/43fdd3164d3648ef63779ad86001d5167580e71e.patch) (468 bytes) - added by Upasana on 2013-04-26 22:13:31Z - Please check this file, not the previos one.
* [bug_6481_3.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3ed/3ed0f159b311e7c68bb73578666d93bb3185bb24/a15152175466f4f5ff4c27bd4abe3c060bee16db.patch) (336 bytes) - added by Upasana on 2013-04-27 17:46:36Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6481 6481
type__enhancement enhancement
reporter__Upasana Upasana
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__easy easy
time__1367013166000000 1367013166000000
changetime__1367089232000000 1367089232000000
version__None None
owner__thijs thijs
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to Upasana|
|-|-|

Thanks for your patch Upasana. As previously discussed in [[#6482](https://github.com/twisted/twisted/issues/6482)#comment:3 ticket:6482, comment #3], the change to the test header is unneccessary and considered a bug in the `twistedchecker` tool. Could you refresh the patch and consider filing a bug for the `twistedchecker` project?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch Upasana, I'll apply it. Make sure that next time you create the patch from the root of your checkout, as described [TwistedDevelopment#SubmittingaPatch here].

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38289]) Apply bug_6481_3.patch: Put the module docstring on 3 lines.

Author: Upasana
Reviewer: thijs
Fixes: [#6481](https://github.com/twisted/twisted/issues/6481)

**Comment by twisted-trac** [comment]
|<img alt="Upasana's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_7"></a>|Upasana commented|
|-|-|

I'll keep this in mind from the next time.
Thanks.
